api/base/renderers.py

```diff
@@ -31,3 +31,14 @@ def get_context(self, *args, **kwargs):
         for form in unwanted_forms:
             del context[form]
         return context
+
+
+class PlainTextRenderer(StaticHTMLRenderer):
+    """
+    Renders plain text.
+
+    Inherits from StaticHTMLRenderer for error handling.
+    """
+
+    media_type = 'text/markdown'
+    format = '.txt'
```

Comment:
I'm not so sure, but it's required according to the docs http://www.django-rest-framework.org/api-guide/renderers/#custom-renderers:
> To implement a custom renderer, you should override BaseRenderer, set the .media_type and .format properties, and implement the .render(self, data, media_type=None, renderer_context=None)